### PR TITLE
[Feature] 12627 enable disable wp details view

### PR DESCRIPTION
**Pull request by hschink**
[`* `#12627` Enable/Disable work package details view`](https://www.openproject.org/work_packages/12627)
[`* `#10190` Clicking work package row does not select the row`](https://www.openproject.org/work_packages/10190)

### CODE REVIEW

File: public/templates/work_packages/work_packages_table.html [hunk 1]
```diff
@@ -102,6 +102,8 @@
         has-context-menu
         target="WorkPackageContextMenu"
         locals="rows, row"
+        single-click="selectWorkPackage(row, $event)"
+        ng-dblClick="showWorkPackageDetails(row.object)"
```
Comment (myabc) [inline public/templates/work_packages/work_packages_table.html]: @hschink does this work with a capitalised C? I think it should be `ng-dblclick`

Comment (hschink) [inline public/templates/work_packages/work_packages_table.html]: @myabc It works for me but I gonna change that.

**Comment by myabc** [comment]
@hschink basic click + double-click behaviours work for me.

I don't think this is specified anywhere, but I would expect the double-click to also select, in addition to opening the details pane.

I'm also seeing some quirky behaviour when doing multiple selections (shift + click)

**Comment by manwithtwowatches** [comment]
@hschink When right clicking it doesn't unselect the other selected rows - as in the last bullet point of the acceptance criteria of https://www.openproject.org/work_packages/12627

**Comment by manwithtwowatches** [comment]
@hschink If a row is checked and then you click on the "i" icon then it opens the details pane but also unchecks the row. I think it would make sense if the row says checked in that case.

**Pull request merged**